pkg/util/child_linux.go

```diff
@@ -0,0 +1,31 @@
+// +build linux
+
+/*
+Derived from:
+
+https://github.com/kubernetes/kubernetes/blob/master/pkg/util/interrupt/interrupt.go
```

Comment:
File name is wrong here too :)
I think omitting is probably the best, we can refer to the package as a whole.